web/discord_relay/queue.go

```diff
@@ -12,6 +12,7 @@ var discord *discordgo.Session
 
 type MessageStruct struct {
 	Realm     string
+	Type 	  string
 	Content   string
 	Avatar    string
 	SteamName string
```

Comment:
This would need to change to account for the new `{ type: "", data: {} }` format

Reply:
How would i do this? i honestly wouldn't know how to do that in go